fetch.bs

```diff
@@ -4419,7 +4452,9 @@ these steps:
    given <var>fetchParams</var>'s <a for="fetch params">cross-origin isolated capability</a>.
 
    <li><p>Set <var>response</var> to the result of invoking <a for=/>handle fetch</a> for
-   <var>requestForServiceWorker</var>. [[!HTML]] [[!SW]]
+   <var>requestForServiceWorker</var>, with <var>fetchParams</var>'s
+   <a for="fetch params">cross-origin isolated capability</a> and <var>fetchParams</var>'s
+   <a for="fetch params">controller</a>. [[!HTML]] [[!SW]]
```

Comment:
What do you think about making the order controller, request, capability? That would match rough order of importance I think.

Reply:
Sure

Reply:
Actually I think it should be `request`, `controller` and `capability`. Request is the direct thing being passed, and the other two are more like context.